sonar-project.properties

```
@@ -3,16 +3,14 @@ sonar.projectKey=hoot
 sonar.exclusions=**/*.pb.cc,**/*.pb.h,**/*.sql,**/*.o,**/*.gcda,**/*.gcno,**/*.qmake,**/*.pro,**/VersionDefines.h,**/ConfigDefaults.h,**/ConfigOptions.h,**/TgsConfig.h
 sonar.github.repository=ngageoint/hootenanny
 sonar.host.url=https://sonarcloud.io
-sonar.sources=./hoot-core/src/main,./hoot-js/src/main,./hoot-rnd/src/main,./tbs/src/main,./tgs/src/main
+sonar.sources=./hoot-core/src/main,./hoot-js/src/main,./tbs/src/main,./tgs/src/main
 sonar.issue.ignore.multicriteria=cout1,cout2,cout3,cout4,cout5,cout6,cout7,cout8,protected1,undef1,empty1,explicit1,override1,ruleOf5,floatcounter,commentedCode,singleDeclaration,literalSuffix,generalCatch,macroParenthesis,misraMacro,cognitiveComplexity,initializer,classSize,structSize,hootLog,tripleNest,forEach,noAuto,namespace,subexpressions,nullptrAuto1,nullptrAuto2,ruleOfZero,usingTypedef1,usingTypedef2,ignoreTodo
```

You removed the rule `cout2` below so it needs to be removed here also.
```suggestion
sonar.issue.ignore.multicriteria=cout1,cout3,cout4,cout5,cout6,cout7,cout8,protected1,undef1,empty1,explicit1,override1,ruleOf5,floatcounter,commentedCode,singleDeclaration,literalSuffix,generalCatch,macroParenthesis,misraMacro,cognitiveComplexity,initializer,classSize,structSize,hootLog,tripleNest,forEach,noAuto,namespace,subexpressions,nullptrAuto1,nullptrAuto2,ruleOfZero,usingTypedef1,usingTypedef2,ignoreTodo
```

---

got it. thanks.